### PR TITLE
Update non inclusive terms where possible

### PR DESCRIPTION
**Pull request by tinder-owenthomas**
Updates all mentions of the default branch in this repo to `main`, as well as a handful of other similar updates.

### CODE REVIEW

**Comment by tinder-owenthomas** [comment]
Hey @thomasvl, you seem to be a pretty frequent contributor here! Please let me know if there's anything I can do to help get this merged. Trying to remove all mentions of non-inclusive terminology from our codebase as well as the third parties we use and this one will help out a lot.

File: Makefile [hunk 1]
```diff
@@ -338,7 +338,7 @@ test-plugin: build ${PROTOC_GEN_SWIFT}
 # against our menagerie of sample protos.
 #
 # If you do this, you MUST MANUALLY verify these files before checking them in,
-# since the new checkin will become the new master reference.
+# since the new checkin will become the new main reference.
```
Comment (Lukasa) [inline Makefile:341]: I think this usage may be better server as “canonical”, not “main”.

**Review by Lukasa — APPROVED** [review]
One minor suggestion

**Comment by thomasvl** [comment]
> Hey @thomasvl, you seem to be a pretty frequent contributor here! Please let me know if there's anything I can do to help get this merged. Trying to remove all mentions of non-inclusive terminology from our codebase as well as the third parties we use and this one will help out a lot.

I'll let one of the Apple folks do signoff/merge since they own the repo and there are changes to the license blocks.

**Review by tbkka — APPROVED** [review]
Looks fine.

**Comment by tbkka** [comment]
@tinder-owenthomas If you could do us the favor of rebasing to main, that should make this easier to follow.

**Comment by tinder-owenthomas** [comment]
@tbkka Sorry, saw your note a couple of hours too late. Looks like it's merged now but if there's anything else you'd like me to do please let me know.

**Pull request merged**